app/graphql/mutations/plans/create.rb

```diff
@@ -18,6 +18,7 @@ class Create < BaseMutation
       argument :name, String, required: true
       argument :parent_id, ID, required: false
       argument :pay_in_advance, Boolean, required: true
+      argument :tax_codes, [String], required: false
```

Comment:
can we have this named `applied_taxes` and be an array of `Types::Taxes::Object`? So I can retrieve the data directly using graphql response instead of performing a call to retrieve tax information such as the rate? 
I don't think we have a `GET /taxes/:id` endpoint